StyleGuide.cshtml

```diff
@@ -13,4 +13,14 @@
 	<li>@thing</li>
 }
 
-</ul>
\ No newline at end of file
+</ul>
+
+@* data tracks should be added to any button on customer facing sites with following structure
+	category-label-action
+snake case should be used for each of the sections if multiple words are needed, and sections 
+are seperated by a dash
+
+button text is within a span*@

```

Comment:
Can you explain the reason behind the span for future peeps?

Reply:
no lol can you?